src/EDGAR/parser.py

```diff
@@ -423,6 +446,62 @@ def _gen_10q_annotated_metadata(
             'is_10q_annotated'] = False
         return False
 
+    def _gen_8k_annotated_metadata(
```

Comment:
I think we should just make a generic ```_get_submission_annotation_metadata(tikr: str, submission: str, document_type: Union[DocumentType, str]) -> bool```

I am not seeing whether this is different than a copy paste of ```gen_10q_annotated_metadata````